frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java

```diff
@@ -204,9 +204,9 @@ private void startWorkerCPP(int port, String runtimeType)
         }
 
         argl.add(cppBackendBin.getAbsolutePath());
-        argl.add("--sock_type");
+        argl.add("--socket_type");
```

Comment:
Revert. This is used in python backend. I will update this in the cpp part in my next pr

Reply:
done